tests/test_backend.py

```diff
@@ -202,6 +200,19 @@ def test_errors(tmpdir, backend):
         file_name,
     )
 
+    with pytest.raises(ValueError):  # extension does not match path
```

Comment:
As you test with `pytest.raises` already could you add a check for the correct error message with `match`, and also for the other tests.

Reply:
done